src/coins.cpp

```diff
@@ -616,13 +616,13 @@ double CCoinsViewCache::GetPriority(const CTransaction &tx, int nHeight) const
     if (tx.IsCoinBase())
         return 0.0;
 
-    // Joinsplits do not reveal any information about the value or age of a note, so we
+    // Shielded transactions do not reveal any information about the value or age of a note, so we
```

Comment:
This should be "Shielded transfers". The transaction might be only partially shielded; in that case the value might be inferrable anyway (e.g. if there are only transparent inputs), but we return max priority for simplicity and because prioritizing by value doesn't actually make a lot of sense in the first place, IMHO.